synapse/lib/cell.py

```diff
@@ -774,6 +774,13 @@ class Cell(s_nexus.Pusher, s_telepath.Aware):
             'type': 'string',
             'hideconf': True,
         },
+        'cell:minspace': {
```

Comment:
trying to minimize the exposure of the term "cell", do we think this belongs to any larger class of limits and would be better grouped as `limit:disk:free` or something a bit more self evident?

Reply:
yeah I could see there being more limits so `limit:disk:free` sounds good